src/node/mod.rs

```diff
@@ -917,25 +918,23 @@ impl Node {
     pub fn parsec_last_version(&self) -> u64 {
         self.stage
             .approved()
-            .map(|stage| stage.parsec_map.last_version())
+            .map(|stage| stage.consensus_engine.parsec_version())
             .unwrap_or(0)
     }
 
     /// Checks whether the given location represents self.
     pub fn in_src_location(&self, src: &SrcLocation) -> bool {
-        self.stage
-            .approved()
-            .map(|stage| stage.chain.in_src_location(src))
-            .unwrap_or(false)
+        src.contains(self.core.name())
     }
 
     /// Returns the prefixes of all our neighbours
     pub fn neighbour_prefixes(&self) -> BTreeSet<Prefix<XorName>> {
-        self.chain()
-            .map(|chain| {
-                chain
-                    .neighbour_infos()
-                    .map(|info| info.prefix())
+        self.shared_state()
+            .map(|state| {
+                state
+                    .sections
+                    .other()
```

Comment:
I also try to avoid using different terms to refer to the same thing but that is not the case here. "other" means a section that is not our section and "neighbour" means a section whose prefix differs from our prefix by at most one bit. So all neighbours are also others, but not the other way around. 

That said, currently other actually contains only neighbours so the two terms seemingly mean the same thing. But that's going to change soon because we are going to keep all sections, not just neighbours and we might stop using the term "neighbour" at all.